sonar-db/src/main/java/org/sonar/db/permission/GroupPermissionDao.java

```diff
@@ -47,22 +51,32 @@ public int countGroups(DbSession session, String permission, @Nullable Long comp
   }
 
   /**
-   * ordered by group names
+   * @return group names, sorted in alphabetical order
+   * @deprecated not compatible with organizations.
    */
+  @Deprecated
   public List<String> selectGroupNamesByPermissionQuery(DbSession dbSession, PermissionQuery query) {
     return mapper(dbSession).selectGroupNamesByPermissionQuery(query, new RowBounds(query.getPageOffset(), query.getPageSize()));
   }
 
+  /**
+   * @deprecated not compatible with organizations.
+   */
+  @Deprecated
   public int countGroupsByPermissionQuery(DbSession dbSession, PermissionQuery query) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1133)